app/src/main/java/org/tahoe/lafs/extension/Extensions.kt

```diff
@@ -83,17 +84,29 @@ fun String.getTokenFromScanUrl(): String {
     return EMPTY
 }
 
-fun String.getActualApiUrl(): String {
+fun String.getActualApiUrl(): URL? {
```

Comment:
We use api level 21 for this app, which doesn't have `Optional` yet. Kotlin's nullable syntax helps here